src/data/builder-tools.js

```diff
@@ -760,6 +760,14 @@ export const Showcases = [
     getstarted: "https://pkg.go.dev/github.com/blinklabs-io/adder",
     tags: ["cli", "golang", "chainindex"],
   },
+  {
+    title: "Cardano Node API",
+    description: "An HTTP API for interfacing with a local Cardano Node and providing the node internal data for HTTP clients.",
+    preview: require("./builder-tools/cardano-node-api.png"),
+    website: "https://github.com/blinklabs-io/cardano-node-api",
+    getstarted: "https://pkg.go.dev/github.com/blinklabs-io/cardano-node-api",
+    tags: ["chainindex", "http", "golang", "json", "websocket"],
```

Comment:
I believe `chainindex` tag might be misplaced, it can be a bridge for chain indexers, but node architecture itself does not provide chain indexing, happy to go with my thumbs-up otherwise

Reply:
I didn't have it originally, but added it after I noticed Ogmios had it. I can remove it.

Reply:
I'll remove it.